packages/cosmic-swingset/lib/ag-solo/vats/vat-http.js

```diff
@@ -40,12 +40,20 @@ export function buildRootObject(vatPowers) {
       const channelID = channelHandleToId.get(channelHandle);
       if (channelID) {
         const o = { ...obj, meta: { channelID } };
-        D(commandDevice).sendBroadcast(o);
+        D(commandDevice).sendBroadcast(JSON.parse(JSON.stringify(o)));
```

Comment:
What does round-tripping through JSON accomplish here?

Reply:
I don't actually think it's necessary anymore.  Removing.